validator/migrations/0028_auto_20210216_2040.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.3 on 2021-02-16 20:40
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('validator', '0027_auto_20210115_1024'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='validationrun',
+            name='scaling_method',
+            field=models.CharField(choices=[('none', 'No scaling'), ('min_max', 'Min/Max'), ('linreg', 'Linear regression'), ('mean_std', 'Mean/standard deviation'), ('cdf_beta_match', 'CDF matching with beta distribution fitting')], default='mean_std', max_length=20),
+        ),
+    ]
```

Comment:
Is this in this PR on purpose? Seems a bit unrelated.

Reply:
no, looks like something that came from the update scaling

Reply:
Ok, it looks like this is not yet in the master branch, so maybe you can just leave it in here.